src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultModelBindingContext.cs

```diff
@@ -130,32 +132,16 @@ public DefaultModelBindingContext()
         }
 
         /// <inheritdoc />
-        public override NestedScope EnterNestedScope()
```

Comment:
Let's do a pass to reorder this class _separately_. This move doesn't do much more than make the review harder to read.